src/js/initializers/initPersistance.js

```diff
@@ -5,7 +5,12 @@ import throttle from "lodash/throttle"
 
 import type {State} from "../state/types"
 
-export const VERSION = "5"
+export const VERSION = "6"
```

Comment:
This version string is just an arbitrary number, not tied to our app versions in any way.

Reply:
Both the global reducer and the window reducer import this same constant.